src/main/java/com/sahabuddin/contactmanager/entities/User.java

```diff
@@ -15,11 +17,17 @@ public class User {
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
+    @NotBlank(message = "Name is required")
```

Comment:
Bhai, it will be better if you use a SignUp request and apply these validation annotation there.